<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startView</class>
 <widget class="QMainWindow" name="startView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>939</width>
    <height>571</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>939</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>快证件照</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>打开照片</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuhelp"/>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
